bedrock/settings/static_media.py

```diff
@@ -349,8 +343,6 @@
     },
     'firefox_fx38_0_5_firstrun': {
         'source_filenames': (
-            'css/sandstone/sandstone.less',
-            'css/tabzilla/tabzilla-static.less',
```

Comment:
I added tabzilla-static.less back to the firefox bundle because this is a shared bundle as referenced within firefox/base.html.

It wasn't inheriting styles correctly because the base was including the firefox bundle inside of the page_css block instead of the site_css block. 

So basically all base.html templates should not be using the page_css block otherwise that would most likely be overwritten by child pages (as it should be).